<launch>
  <!-- node name="hokuyo" pkg="hokuyo_node" type="hokuyo_node" respawn="false" output="screen" -->
    <!-- Starts up faster, but timestamps will be inaccurate. --> 
    <!-- param name="calibrate_time" type="bool" value="false"/ --> 
    <!-- Set the port to connect to here -->
    <!-- param name="port" type="string" value="/dev/ttyACM0"/> 
    <param name="intensity" type="bool" value="false"/>
  </node -->
  
  <node pkg="mp_tracker" type="d_tracker_plus_node" name="phd_laser_tracker" output="screen" >
    <param name="tracking_frame" value="/map"/>
    <param name="laser_topic" value="/bender/sensors/laser_front/scan"/>
    <param name="pointcloud_topic" value="/openni/depth/points"/>
    <param name="camera_topic" value="/openni/rgb/image_color"/>
    <param name="camera_info_topic" value="/openni/rgb/camera_info"/>
    <param name="use_beta_mixture" value="false"/>
    <param name="use_optimal_beta_merge" value="false"/>
    <param name="merge_beta_threshold" value="0.04"/>
    <param name="merge_gauss_threshold" value="0.1"/>
    <param name="birth_prior_weight" value="0.05"/>
    <param name="max_components" value="200"/>
    <param name="clutter" value="0.5"/> <!-- falsos positivos por scan, subirlo -->
    <param name="use_leg_detector" value="true"/>
    <param name="debug_script" value=""/>
    <param name="sigma_obs" value="0.05"/> 

  </node>


  <!-- node name="rviz" pkg="rviz" type="rviz" respawn="false" output="screen"/>
  
  <node pkg="tf" type="static_transform_publisher" name="odom_to_map" args="0 0 0 0 0 0 /odom /map 100" />
  <node pkg="tf" type="static_transform_publisher" name="map_to_laser" args="0 0 0 0 0 0 /map /laser 100" / -->
</launch>
